src/sentry/static/sentry/app/views/settings/components/forms/controls/radioGroup.jsx

```diff
@@ -70,13 +71,14 @@ const RadioLineButtonFill = styled.div`
   border-radius: 50%;
   background-color: ${p => p.theme.green};
   animation: ${p => (p.animate ? `0.2s ${growIn} ease` : 'none')};
-  opacity: ${p => (p.isDisabled ? 0.4 : null)};
+  opacity: ${p => (p.disabled ? 0.4 : null)};
 `;
 
 const RadioLineText = styled.div`
   margin-left: 0.5em;
   font-size: 0.875em;
   font-weight: bold;
+  opacity: ${p => (p.disabled ? 0.4 : null)};
```

Comment:
I believe this will now produce DOM warnings, since emotion will pass `isDisabled` down to the DOM element, which divs cannot be disabled.